### ISSUE TITLE
`quiet` config option doesn't silence logs for `truffle test`

### ISSUE THREAD

**Issue by sbrichards** [trufflesuite/truffle#2238]
- [x] I've asked for help in the [Truffle Gitter](http://gitter.im/Consensys/truffle) before filing this issue.

---------------------------

## Issue

Truffle config provides a nice feature to silence compilation logs. If I add:
```
module.exports = {
  quiet:  true
}
```
To my config and run `truffle compile`, this works as expected. But when I run `truffle test`, truffle still outputs all compilation logs. I expected `truffle test` to also respect my quiet config.

## Steps to Reproduce

- Add the `quiet` flag to the config
- Run `truffle test`

## Expected Behavior

I expected Truffle to silence the compile logs.

## Actual Results

Full output logs of my project when running `truffle test`:
https://gist.github.com/sbrichards/e057c51042bf509082bba35cb95d5498

## Environment

* Operating System: macOS Mojave 10.14.5.
* Ethereum client: Ganache
```
$ truffle version
Truffle v5.0.21 (core: 5.0.21)
Solidity - 0.4.24 (solc-js)
Node v10.13.0
Web3.js v1.0.0-beta.37
```

**Comment by sbrichards** [comment]
FYI @CruzMolina - here's the issue we discussed. Thanks in advance!

**Comment by gnidan** [comment]
This should be released in Truffle v5.0.29! Try it out and let us know. Thanks!
